<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string comment="Analyzes IP traffic" >IPanalyzer</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/images.qrc" >:/net/preferences-system-network.png</iconset>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>1</number>
    </property>
    <property name="topMargin" >
     <number>2</number>
    </property>
    <property name="rightMargin" >
     <number>1</number>
    </property>
    <property name="bottomMargin" >
     <number>2</number>
    </property>
    <item row="0" column="0" >
     <widget class="QListView" name="listView" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice" >
    <property name="title" >
     <string>Device</string>
    </property>
    <addaction name="actionStart" />
    <addaction name="actionStop" />
   </widget>
   <widget class="QMenu" name="menuConnection" >
    <property name="title" >
     <string>Connection</string>
    </property>
    <addaction name="actionAnalyze" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOptions" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuDevice" />
   <addaction name="menuConnection" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart" />
   <addaction name="actionStop" />
  </widget>
  <action name="actionStart" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../res/images.qrc" >:/control/media-playback-start.png</iconset>
   </property>
   <property name="text" >
    <string>Start Capture</string>
   </property>
   <property name="statusTip" >
    <string>Starts capture on selected interface</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../res/images.qrc" >:/control/media-playback-stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop Capture</string>
   </property>
   <property name="statusTip" >
    <string>Stops capture on selcted interface</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAnalyze" >
   <property name="icon" >
    <iconset resource="../res/images.qrc" >:/net/dialog-information.png</iconset>
   </property>
   <property name="text" >
    <string>Analyze</string>
   </property>
  </action>
  <action name="actionOptions" >
   <property name="icon" >
    <iconset resource="../res/images.qrc" >:/file/system-run.png</iconset>
   </property>
   <property name="text" >
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="../res/images.qrc" >:/file/application-exit.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
